README.md

```diff
@@ -219,6 +219,11 @@ Since for streaming responses OpenAI doesn't supply the full usage tokens summar
 
 NOTE: Stream is currently only supported with SDK usage, and not with using REST directly.
 
+### Langchain support
+
+We currently only support monitoring OpenAI endpoints when using Langchain via monkey-patching the openai.Completion or openai.ChatCompletion before creating the LLM object. See completion_langchain_monkeypatch.py for an example.
```

Comment:
The first sentence is a bit too long and hard to understand. Try to rephrase/add punctuation marks